src/accordian/Accordion.tsx

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {classNames} from "..//utils";
+import {ClassNames} from "./ClassNames";
+import {Icon, Direction} from "../icon";
+
+type AccordianProps = {
+    style?: any;
+    className?: string;
+    content: accordionContent[];
+    accordionMultiPanel?: boolean;
+};
+
+type accordionContent = {

```

Comment:
Is it possible to add user defined style and classname to item and content?

Reply:
Addressed - Setting up type of title as React-node so that we can pass title as HTML with styling attached to it. For content it is already working.